### ISSUE TITLE
Console output format needs improvement

### ISSUE THREAD

**Issue by fordaj** [fordaj/lowpy#12]
Console output should include:

- [x] Model description by layer (learning rate, layer dimensions, etc.)
- [x] Table-based output showing epochs, iteration, test number, and training/testing accuracy/loss
